docker/simplified_app.sh

```diff
@@ -53,12 +58,17 @@ SIMPLIFIED_ENVIRONMENT=/var/www/circulation/environment.sh
 echo "if [[ -f $SIMPLIFIED_ENVIRONMENT ]]; then \
       source $SIMPLIFIED_ENVIRONMENT; fi" >> env/bin/activate
 
+# Install Poetry
+curl -sSL https://install.python-poetry.org | POETRY_HOME="/opt/poetry" python3
```

Comment:
In the future it might worth it to use multi-stage build to save the target image size by getting `poetry` out of it.